example/main.go

```diff
@@ -6,9 +6,9 @@ import (
 	"crypto/rsa"
 	"crypto/x509"
 	"encoding/pem"
-	"errors"
 	"fmt"
 	"os"
+
```

Comment:
`[nit]`

Usually we don't leave an empty line at the end of the `import` list. So you can remove it.